rules/windows/execution_downloaded_shortcut_files.toml

```diff
@@ -0,0 +1,59 @@
+[metadata]
+creation_date = "2020/09/02"
+ecs_version = ["1.6.0"]
+maturity = "development"
+query_schema_validation = false
+updated_date = "2020/09/02"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies .lnk shortcut file downloaded from outside the local network. These shortcut files are commonly used in
+phishing campaigns.
+"""
+index = ["logs-endpoint.events.*"]
```

Comment:
```suggestion
index = ["logs-endpoint.events.file-*"]
```

Reply:
We've been adding `logs-endpoint.events.*` for all endpoint based rules. I don't think its worth it to require them to create another index for just a few rules.

Reply:
good point on balance here: It is more efficient but then there is the additional expectation on users to have the index created for each subset of events